sdk/storage/Microsoft.Azure.WebJobs.Extensions.Storage.Queues/src/Microsoft.Azure.WebJobs.Extensions.Storage.Queues.csproj

```diff
@@ -4,8 +4,6 @@
   </PropertyGroup>
   <PropertyGroup>
     <Version>5.1.0-beta.1</Version>
-    <!--The ApiCompatVersion is managed automatically and should not generally be modified manually.-->
-    <ApiCompatVersion>5.0.1</ApiCompatVersion>
```

Comment:
I don't think we want to remove the ApiCompatVersion from Blobs/Queues - only from the Storage csproj.